openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -103,14 +122,14 @@ def _verify_private_account_response(self, response):
         data = response.data
         self.assertEqual(2, len(data))
         self.assertEqual(self.user.username, data["username"])
-        self.assertIsNone(data["profile_image"])
+        self._verify_profile_image_data(data)
 
     def _verify_full_account_response(self, response):
         """
         Verify that all account fields are returned (even those that are not shareable).
         """
         data = response.data
-        self.assertEqual(12, len(data))
+        self.assertEqual(13, len(data))
         self.assertEqual(self.user.username, data["username"])
         self.assertEqual(self.user.first_name + " " + self.user.last_name, data["name"])
```

Comment:
Add check of profile image as well. Perhaps update "mock profile" method so it ensures that the user DOES have a custom profile image?